package.xml

```diff
@@ -0,0 +1,23 @@
+<package id      = "junk_search"
```

Comment:
- This is up one directory from where it should be.
- Didn't I do this already?